lib/inc/cpp-pcp-client/connector/connector.hpp

```diff
@@ -159,8 +159,8 @@ class LIBCPP_PCP_CLIENT_EXPORT Connector {
     ///
     /// Throws a connection_not_init_error in case the connection has
     /// not been opened previously.
-    /// Also rethrows any exception that can be thrown by
-    /// startMonitoring.
+    /// Rethrows a possible exception previously caught by the
```

Comment:
In case of an exception, the Monitor Task, i.e. `startMonitoring`, may filter or store it in `monitor_exception_`, then the dtor or `stopMonitoring` will rethrow it.
I forgot to update the `startMonitoring` docstring, as commented in #182.